### PR TITLE
updating operator and OLM bits for 0.11.0 community release

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: deploy/olm-catalog/knative-eventing-operator/0.10.0/knative-eventing-operator.v0.10.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@ spec:
                   value: knative-eventing-operator
                 image: quay.io/openshift-knative/knative-eventing-operator:v0.10.0
                 args:
-                  - --filename=https://raw.githubusercontent.com/openshift/knative-eventing/release-v0.10.0/openshift/release/knative-eventing-v0.10.0.yaml
+                  - --filename=https://raw.githubusercontent.com/openshift/knative-eventing/release-v0.10.0/openshift/release/knative-eventing-v0.11.0.yaml
```
Comment (aliok) [inline deploy/olm-catalog/knative-eventing-operator/0.10.0/knative-eventing-operator.v0.10.0.clusterserviceversion.yaml]:
```suggestion
                  - --filename=https://raw.githubusercontent.com/openshift/knative-eventing/release-v0.11.0/openshift/release/knative-eventing-v0.11.0.yaml
```

Comment (matzew) [reply deploy/olm-catalog/knative-eventing-operator/0.10.0/knative-eventing-operator.v0.10.0.clusterserviceversion.yaml]: @aliok that is not correct - that is actually the override for the 0.10.0 image ;-)

Comment (matzew) [reply deploy/olm-catalog/knative-eventing-operator/0.10.0/knative-eventing-operator.v0.10.0.clusterserviceversion.yaml]: that thing should not be 0.11 ;-) I will fix the second part

**Pull request merged**